src/types/shims.d.ts

```
@@ -0,0 +1,2 @@
+declare module 'luxon';
```

ESLint

[error] Replace `'luxon'` with `"luxon"`

(prettier/prettier)